### PR TITLE
Add OrgasmicNightmare/cakephp-seo plugin

### PR DESCRIPTION
**Pull request by orgasmicnightmare**
A SEO plugin for cakePHP 3 to manage seo tags with ease. It comes with a Behavior, a Component, a View Cell and the default admin crud actions for each tables.

The behavior helps you to generate some default tags for each model. You can use simple text, pattern, callbacks.
The component inject tags in the view.
The view cell helps you to add or edit tags directly in the admin view of a record.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,6 +275,7 @@ Additional lists you might find useful:
 *Search Engine Optimization.*
 
 - [Muffin/Slug plugin](https://github.com/UseMuffin/Slug) - A plugin for generating slugs and finding records by slug. Uses a pluggable architecture which allows using your own slug generator class.
+- [OrgasmicNightmare/cakephp-seo](https://github.com/orgasmicnightmare/cakephp-seo) - Manage seo tags with ease. It comes with a Behavior, a Component, a View Cell and the default admin crud actions for each tables.
```
Comment (dereuromark) [inline README.md]: Please respect the guidelines for the name.

**Comment by orgasmicnightmare** [comment]
Oh my bad, It should be ok.

File: README.md (hunk 1 above)
Comment (orgasmicnightmare) [inline README.md]: Oh my bad, It should be ok.

File: README.md [hunk 2]
```diff
@@ -275,6 +275,7 @@ Additional lists you might find useful:
 *Search Engine Optimization.*
 
 - [Muffin/Slug plugin](https://github.com/UseMuffin/Slug) - A plugin for generating slugs and finding records by slug. Uses a pluggable architecture which allows using your own slug generator class.
+- [Seo plugin](https://github.com/orgasmicnightmare/cakephp-seo) - Manage seo tags with ease. It comes with a Behavior, a Component, a View Cell and the default admin crud actions for each tables.
```
Comment (dereuromark) [inline README.md]: crud => CRUD imo.
But I wonder if you need to be that specific regarding Behavior, Component etc.
I think the description should not list the exact types of classes used, but the functionality this plugin does.

**Pull request merged**